components/automate-cds/server/root.go

```diff
@@ -40,26 +41,42 @@ func (s *Server) ListContentItems(ctx context.Context, request *request.ContentI
 	return &response.ContentItems{
 		Items: []*response.ContentItem{
 			{
-				Name:        "DevSec Linux Security Baseline Benchmark Level 1",
-				Description: "Test suite for best practice Linux OS hardening",
-				Type:        "profile",
-				Version:     "1.9.0",
-				Platform:    "",
+				Id:             "f354577e-2ea5-4c96-86ed-78a8ba68e0f3",
+				Name:           "DevSec Linux Security Baseline Benchmark Level 1",
+				Description:    "Test suite for best practice Linux OS hardening",
+				Type:           "profile",
+				Version:        "1.9.0",
+				Platforms:      []string{"Ubuntu", "Debian"},
+				CanBeInstalled: true,
 			},
 			{
-				Name:        "Audit Benchmark level 1",
-				Description: "Test suite for best practice Linux OS hardening",
-				Type:        "cookbook",
-				Version:     "9.2.1",
-				Platform:    "",
+				Id:             "1f9e6a1e-3381-487f-98ef-9966c65ff92a",
+				Name:           "Audit Benchmark level 1",
+				Description:    "Test suite for best practice Linux OS hardening",
+				Type:           "cookbook",
+				Version:        "9.2.1",
+				Platforms:      []string{"Redhat", "Centos"},
+				CanBeInstalled: false,
 			},
 			{
-				Name:        "Compliance Effortless Package",
-				Description: "This is a hart file that enables effortless compliance on your fleet.",
-				Type:        "package",
-				Version:     "2.3.2",
-				Platform:    "",
+				Id:             "736f5724-b384-4d74-b8cb-dedd9f1ecd54",
+				Name:           "Compliance Effortless Package",
+				Description:    "This is a hart file that enables effortless compliance on your fleet.",
+				Type:           "package",
+				Version:        "2.3.2",
+				Platforms:      []string{"Solaris"},
+				CanBeInstalled: true,
 			},
 		},
 	}, nil
 }
+
+// InstallContentItem - installing a content item
+func (s *Server) InstallContentItem(ctx context.Context, request *request.InstallContentItem) (*response.InstallContentItem, error) {
+
+	log.Infof("Downloading content item with ID %s ...", request.Id)
+
+	log.Infof("Installing content item with ID %s ...", request.Id)
+
+	return &response.InstallContentItem{}, nil
+}
```

Comment:
Currently, we only log a message when a user request for an item to be installed.